src/superio/nuvoton/nct5104d/nct5104d.h

```diff
@@ -23,6 +23,7 @@
 #define IRQ_TYPE_SEL_CR10   0x10     /* UARTA,UARTB */
 #define IRQ_TYPE_SEL_CR11   0x11     /* SMI,UARTC,UARTD,WDTO */
 #define GLOBAL_OPTION_CR26  0x26
+#define GLOBAL_OPTION_CR7   0x07
```

Comment:
@piotr-kleins this is not needed. Register CR07 is defined 4 lines earlier as LDN_SELECT.